oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Thank you for wanting to avoid adding length to `client.py`.

That you defined these as functions and provided independent test coverage suggests that to you they are logically and semantically independent of the one-each use-sites of them that you also provide. That alone strikes me as reason to keep them in their own module; I don't consider their short length material in any way.

@jonparrott is absolutely right that we are trying our hardest to keep the `oauth2client`'s public API small. I don't often cite the library's mission statement, but if these functions were library-public, would they fit in with "This is a Python library for accessing resources protected by OAuth 2.0."?